LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -144,24 +144,15 @@ class LockboxXCUITests: BaseTestCase {
         waitforExistence(app.buttons["finish.button"], timeout: 10)
         app.buttons["finish.button"].tap()
         waitforExistence(app.navigationBars["firefoxLockbox.navigationBar"])
-    }
-
-    func test5SendUsageDataSwitch() {
-//        navigator.goto(Screen.SettingsMenu)
-//        // Disable the send usage data
-//        navigator.performAction(Action.SendUsageData)
-//        XCTAssertEqual(app.switches["sendUsageData.switch"].value as? String, "0")
-//
-//        // Enable it again
-//        navigator.performAction(Action.SendUsageData)
-//        XCTAssertEqual(app.switches["sendUsageData.switch"].value as? String, "1")
+        sleep(5)
     }
 
     func test6SortEntries() {
         navigator.goto(Screen.LockboxMainPage)
         waitforExistence(app.navigationBars["firefoxLockbox.navigationBar"], timeout: 10)
         // Checking if doing the steps directly works on bb
         waitforExistence(app.buttons["sorting.button"])
+        sleep(4)
```

Comment:
Same here. Or in this case you could add a `waitforExistence(app.buttons["sorting.button"], timeout: 3)` since the button is not shown until the logins appear

Reply:
Adding waitforExistence(app.buttons["sorting.button"], timeout: 3).